help/markdown/core-process.md

```diff
@@ -1,6 +1,9 @@
 # Starting processes in "FAKE - F# Make"
 
-**Note:  This documentation is for FAKE 5! **
+<div class="alert alert-info">
```

Comment:
Shouldn't we update the css for `**`? (Sorry I don't have any idea what's the best way)

Reply:
I use both alert info and alert warning. Restyling bold everywhere is probably not a great idea. DocFx markdown has an extension to use notes and warnings. I could add that to FSFormatting in theory i guess.